api/v1/api.go

```diff
@@ -0,0 +1,85 @@
+package api
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"go.uber.org/zap"
+)
+
+func path(endpoint string) string {
+	return fmt.Sprintf("/api/v1%s", endpoint)
+}
```

Comment:
string should be a constant 
```suggestion
const baseUrl = "/api/v1"

func path(endpoint string) string {
	return fmt.Sprintf("%s%s", baseUrl, endpoint)
}
```